analysis_options.yaml

```diff
@@ -208,8 +220,11 @@ linter:
     - unnecessary_string_interpolations
     - unnecessary_this
     - unnecessary_to_list_in_spreads
+    # - unnecessary_unawaited
```

Comment:
why not keep this check on unnecessary unawaited?

Reply:
This gives me the warning that it is not a recognized lint rule.
Could be the dart version?

Reply:
oh yeah maybe it's not yet released or something